### ISSUE TITLE
Flag passed to VMware Tools installer may need to be dynamic

### ISSUE THREAD

**Issue by geerlingguy** [geerlingguy/ansible-role-packer-debian#9]
The current command to install VMware Tools is:

```
- name: Run the VMware tools installer.
  shell: /tmp/vmfusion-archive/vmware-tools-distrib/vmware-install.pl --default
```

But for newer versions (8+?) of VMware (and maybe only on more recent OSes), the installation may need `--force-install` instead. See these two commits in Bento:
- https://github.com/chef/bento/commit/4591252939fc6cdb24dc80df05b2835ff84546d6
- https://github.com/chef/bento/blob/d480f3a29441a51687944e4a76ce891a3c41b795/scripts/common/vmtools.sh

**Comment by geerlingguy** [comment]
Or maybe no install at all on 16.04? Trying to suss out what actually happened in Bento upsteam...

**Comment by geerlingguy** [comment]
See also: [Installing VMware tools on a virtual machine that supports open-vm-tools](https://kb.vmware.com/selfservice/microsites/search.do?language=en_US&cmd=displayKC&externalId=2107676)

**Comment by it-praktyk** [comment]
The related issue https://github.com/geerlingguy/packer-ubuntu-1404/issues/21

**Comment by it-praktyk** [comment]
The fix is included in the #13

**Comment by geerlingguy** [comment]
Thanks!

**Comment by it-praktyk** [comment]
The issue can be closed now because is resolved by #15
